prime-angular-frontend/src/app/shared/components/forms/site-information-form/site-information-form.component.html

```diff
@@ -32,50 +32,35 @@
   </app-page-section>
 
   <app-page-section>
-    <div *ngIf="!isCommunityPharmacy()"
-         class="mb-3">
-      <mat-slide-toggle [formControl]="isNewSite"
-                        color="primary">
-        This is a new Site without a Site ID
-      </mat-slide-toggle>
-    </div>
-
-    <ng-container *ngIf="isNewSite.value && !isCommunityPharmacy(); else notNewSite">
-      If this is a new Site you will receive your Site ID upon approval.
-    </ng-container>
-
-    <ng-template #notNewSite>
-      <app-page-subheader2>
-        <ng-container appPageSubheader2Title>Site ID</ng-container>
-        <ng-container appPageSubheader2Summary>
-          Provide the sites
-          ID{{ isCommunityPharmacy() ? '.' : ' or PEC Code. This is a 3-letter code. For example, JZY' }}
-        </ng-container>
-      </app-page-subheader2>
+    <app-page-subheader2>
+      <ng-container appPageSubheader2Title>Site ID</ng-container>
+      <ng-container appPageSubheader2Summary>
+        Provide the site's ID.
+      </ng-container>
```

Comment:
Weird I was basing it off this, since I thought it was from the mock-up. I'll take a look into it to see what one is correct.
![image](https://user-images.githubusercontent.com/59669288/153920879-97ddbdf5-d910-4b48-86e9-51650fc5a024.png)

Reply:
Though that did make me realize i missed the (optional) tag so ill add that too!

Reply:
Following up for the "This is a 3-letter code. For example, XYZ" Since that was only logic for CommPharm and we no longer have any validation that matches that format. Will see if its just leftover from old mock-ups

Reply:
Alright replaced the code to have the text show up for community pharmacy, we will talk about if we need to enforce validation in that case in refinement

Reply:
validation if needed will be another ticket

Reply:
When I choose "Community Pharmacy" care setting, I only see 
![image](https://user-images.githubusercontent.com/7586665/154165899-4a33e692-1158-4046-ad9b-b2d8bcf9ed1c.png)

Is that correct?